projects/fal/src/fal/_serialization.py

```diff
@@ -76,26 +32,20 @@ def add_serialization_listeners_for(obj):
 
     if "." in module_name:
         package_name, *_ = module_name.partition(".")
-        _PACKAGES.add(package_name)
+        cloudpickle.register_pickle_by_value(__import__(package_name))
 
 
-@mainify
-def patch_pydantic_field_serialization():
+def _patch_pydantic_field_serialization() -> None:
```

Comment:
this code actually comes from here where they register custom cloudpickle serializers https://github.com/ray-project/ray/blob/842bbcf4236e41f58d25058b0482cd05bfe9e4da/python/ray/_private/pydantic_compat.py#L80